packages/babel-plugin-polyfill-corejs3/src/index.ts

```diff
@@ -291,7 +288,7 @@ export default defineProvider<Options>(function(
           resolved.desc,
           resolved.name,
           utils,
-          // $FlowIgnore
+          // @ts-ignore
           meta.object,
```

Comment:
this statement and the similar one below it could be refactored with some type narrowing to remove the `// @ts-ignore`, e.g.:

```suggestion
          "object" in meta ? meta.object : undefined,
```

Reply:
It's ok with the suppression comment (I actually prefer it). However, it's better to use `@ts-expect-error` so that if one day the TS compiler becomes smart enough that it's not needed we get "notified" that the comment is not needed anymore.

Reply:
Ah I forgot about that feature!
Though I'm not sure if TS would ever try changing that since it's a safety thing that JS doesn't have